gui/GameConqueror.py

```diff
@@ -554,6 +560,13 @@ def SelectProcess_Button_clicked_cb(self, button, data=None):
     # customed callbacks
     # (i.e. not standard event names are used)
 
+    # Callback to strip user input ends of whitespace.
+    def entry_changed_event_cb(self, entry, data=None):
+        if self.scan_data_type_combobox.get_active_text()!="string":
+            buf = entry.get_text().strip('\n')
+            entry.set_text(buf)
+            entry.set_position(-1)
```

Comment:
I really dislike this, for several reasons:
* '\n' is not special, why not banning '\r', too? and '\b'?
* You get inconsistencies if you switch from string, where '\n' is allowed
* You gain nothing in safety, there is still a parser to run

Either you go all the way and disallow anything that's not a number, math operator, parenthesis, scanmem operator or you let the user put in anything